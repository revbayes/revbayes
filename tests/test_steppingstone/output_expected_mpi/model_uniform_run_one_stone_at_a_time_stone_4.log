Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21553.41	-21542.09	-11.31752	0.9891611	0.105084	0.139066	0.20157	0.106968	0.0851234	0.175028	0.193143	0.103833	0.135953	0.134269	0.194022	0.279648	0.142505	0.144048	0.13401	0.112392	0.0799196	0.0907661	0.118299	0.141697	0.181571	0.113915	0.468232	0.161453	0.0554844	0.0321395	0.0613482	0.110775	0.116226	0.229055	0.0816853	0.125605	0.0426107	0.146636	0.10581	0.280824	0.147379	0.0763847	0.00527101	0.0309938	0.0882787	0.106682	0.212097	0.0375309	0.289797	0.0594208	0.0453779	0.529183	0.0386901	0.134786	0.47347	0.998181	2.39356	0.364293	0.314109	0.0884166	0.233182	0.3330521	5.7978
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
10	-21689.81	-21684	-5.817916	1.207754	0.180159	0.127137	0.117908	0.0550114	0.0290906	0.0652202	0.244224	0.0985352	0.188963	0.066411	0.0808149	0.120917	0.200213	0.178986	0.194845	0.104909	0.122812	0.112634	0.0796371	0.121329	0.109911	0.197283	0.534933	0.0955516	0.222137	0.0154312	0.0461044	0.0391087	0.11446	0.305877	0.115521	0.155084	0.039066	0.073092	0.0643238	0.112091	0.110702	0.0561921	0.0242337	0.114199	0.0495952	0.0965652	0.081618	0.0377729	0.353053	0.052485	0.040092	0.481105	0.0354918	0.176455	0.531385	1.0274	2.26476	0.320176	0.365495	0.0809472	0.233382	0.4919418	5.262836
