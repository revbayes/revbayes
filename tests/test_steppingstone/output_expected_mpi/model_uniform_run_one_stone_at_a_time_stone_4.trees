Iteration	Posterior	Likelihood	Prior	psi
0	-21400.01	-21388.35	-11.65655	(Tarsius_syrichta[&index=21]:0.301722,(((Callicebus_donacophilus[&index=2]:0.188283,((Aotus_trivirgatus[&index=1]:0.127948,Saimiri_sciureus[&index=20]:0.273105)[&index=26]:0.008870,Cebus_albifrons[&index=3]:0.153032)[&index=33]:0.078293)[&index=43]:0.204302,(Hylobates_lar[&index=9]:0.092272,(Pan_paniscus[&index=17]:0.111349,((Chlorocebus_aethiops[&index=5]:0.114517,Macaca_mulatta[&index=13]:0.130827)[&index=25]:0.114195,Colobus_guereza[&index=6]:0.231774)[&index=42]:0.153341)[&index=32]:0.067378)[&index=30]:0.238214)[&index=31]:0.176543,(Daubentonia_madagascariensis[&index=7]:0.204068,((Lepilemur_hubbardorum[&index=11]:0.220111,((Cheirogaleus_major[&index=4]:0.070131,Microcebus_murinus[&index=14]:0.146977)[&index=35]:0.071634,((Lemur_catta[&index=10]:0.141864,Varecia_variegata_variegata[&index=22]:0.154062)[&index=40]:0.025505,Propithecus_coquereli[&index=19]:0.207623)[&index=39]:0.017806)[&index=38]:0.018755)[&index=27]:0.131437,(((Otolemur_crassicaudatus[&index=16]:0.027986,Galago_senegalensis[&index=8]:0.232390)[&index=24]:0.084409,(Nycticebus_coucang[&index=15]:0.191165,Loris_tardigradus[&index=12]:0.162774)[&index=37]:0.083009)[&index=29]:0.082668,Perodicticus_potto[&index=18]:0.190534)[&index=41]:0.053581)[&index=34]:0.027000)[&index=28]:0.055152)[&index=36]:0.083704,Galeopterus_variegatus[&index=23]:0.321198)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
10	-21481.05	-21470.33	-10.72106	((Tarsius_syrichta[&index=21]:0.311245,(Daubentonia_madagascariensis[&index=7]:0.283948,((((Propithecus_coquereli[&index=19]:0.155184,(Varecia_variegata_variegata[&index=22]:0.163235,Lemur_catta[&index=10]:0.076409)[&index=40]:0.034633)[&index=39]:0.027264,(Cheirogaleus_major[&index=4]:0.147424,Microcebus_murinus[&index=14]:0.208545)[&index=35]:0.073773)[&index=38]:0.099682,Lepilemur_hubbardorum[&index=11]:0.069634)[&index=27]:0.084750,(((Nycticebus_coucang[&index=15]:0.204684,Loris_tardigradus[&index=12]:0.133254)[&index=37]:0.033982,Perodicticus_potto[&index=18]:0.320924)[&index=41]:0.033740,(Galago_senegalensis[&index=8]:0.082501,Otolemur_crassicaudatus[&index=16]:0.083514)[&index=24]:0.064048)[&index=29]:0.156786)[&index=28]:0.123633)[&index=34]:0.081946)[&index=36]:0.107669,((((Macaca_mulatta[&index=13]:0.143468,(Colobus_guereza[&index=6]:0.380023,Chlorocebus_aethiops[&index=5]:0.077179)[&index=42]:0.062485)[&index=25]:0.154320,Pan_paniscus[&index=17]:0.072488)[&index=32]:0.048903,Hylobates_lar[&index=9]:0.071237)[&index=30]:0.343265,(Callicebus_donacophilus[&index=2]:0.078884,(Aotus_trivirgatus[&index=1]:0.108848,(Cebus_albifrons[&index=3]:0.126216,Saimiri_sciureus[&index=20]:0.191608)[&index=26]:0.004924)[&index=33]:0.068965)[&index=43]:0.205659)[&index=31]:0.152716,Galeopterus_variegatus[&index=23]:0.274362)[&index=44];
