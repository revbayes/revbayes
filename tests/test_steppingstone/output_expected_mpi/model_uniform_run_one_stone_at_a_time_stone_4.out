state	power	likelihood
4	0.07776	-21665.8
5	0.07776	-21590.5
6	0.07776	-21639
7	0.07776	-21661.1
8	0.07776	-21684.4
9	0.07776	-21715.6
10	0.07776	-21684
